Sources/TCALogsSheetKit/Examples/AnyReducer/_CounterReducer.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  _CounterReducer.swift
+//  
+//
+//  Created by Yermukhanbet Yessen on 2023/01/02.
+//
+
+import Foundation
+import ComposableArchitecture
+
+struct CounterEnvironment { }
```

Comment:
❓ do we need this?

Reply:
oh yes, there is no need!
I will remove it. 🔥